tlog-tiles.md

```diff
@@ -46,8 +46,9 @@ with `Content-Type: text/plain; charset=utf-8`.
 This endpoint is mutable, so its headers SHOULD prevent caching beyond a few
 seconds.
 
-The checkpoint MUST carry at least one Ed25519 signature by the log, and MAY
-carry additional signatures of other types.
+If the log is public or anyway interacting with the public witness network, the
```

Comment:
nit: "If the log is public, or is interacting in any way with the public witness network, ..."